src/components/FeedPage.vue

```diff
@@ -85,6 +98,19 @@ export default {
                 this.loadMoreVideos();
             }
         },
+        shouldShowVideo(video) {
+            switch (this.selectedFilter) {
+                case "all":
+                    return true;
+                case "videos":
+                    return !video.isShort;
+                default:
+                    return video.isShort;
```

Comment:
I'd keep it like this:
```
                case "shorts":
                    return video.isShort;
                case "videos":
                    return !video.isShort;
                default:
                    return true;
```

We could also convert the query string/preference to lowercase too above.